packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1687,8 +1688,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 summaryLogger,
             });
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName] as ISummaryTree;
+            let all = 0;
+            let unreferenced = 0;
```

Comment:
@agarwal-navin Correct me if I am wrong. These are unreferencedSummarizedDataStoreCount because we don't have this unreferenced info for handles(data stores which are not summarized).

Reply:
Yes, you are right. As per my other comment, we need to be clear on what information we want to log.